govdelivery/tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+from test_api import *
```

Comment:
In Python 3 this kind of relative import no longer works, you need to do something like:

```suggestion
from .test_api import *
```

Reply:
The other two lines below this also need to be modified similarly.

Reply:
My kingdom for multi-line suggestions :(